core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/file/UserFileResource.scala

```diff
@@ -16,17 +18,33 @@ import java.util
 import javax.servlet.http.HttpSession
 import javax.ws.rs._
 import javax.ws.rs.core.{MediaType, Response}
+import scala.collection.JavaConverters._
+import scala.collection.mutable
 
 /**
   * Model `File` corresponds to `core/new-gui/src/app/common/type/user-file.ts` (frontend).
   */
+case class fileRecord(
```

Comment:
This name is also collapsed with `edu.uci.ics.texera.web.model.jooq.generated.tables.records.FileRecord` already. We need to change the name.

Reply:
changed.